lib/sort_n_params/concerns/scopes.rb

```diff
@@ -4,15 +4,17 @@ module SortNParams
   module Scopes
     extend ActiveSupport::Concern
     class_methods do
-      def sorting_order(ordering_array)
+      def sorting_order(ordering_array, table_name)
```

Comment:
Could you change this to `table_name = nil`? I think that it has to be an optional param